package/boot/arm-trusted-firmware-mvebu/Makefile

```diff
@@ -8,15 +8,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=arm-trusted-firmware-mvebu
-PKG_VERSION:=2.2
-PKG_RELEASE:=2
+PKG_VERSION:=2.3
+PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_RELEASE).tar.xz
 PKG_SOURCE_URL:=https://git.trustedfirmware.org/TF-A/trusted-firmware-a.git
 PKG_SOURCE_DATE:=2020-01-21
```

Comment:
Please update the date to the date of the commit., in this case 2020-04-20

Reply:
Thanks, I opted for switching to release tarballs, we don't need some specific git commit between releases.